docs/examples/opencensus-shim/app.py

```diff
@@ -18,7 +18,7 @@
 from opencensus.ext.flask.flask_middleware import FlaskMiddleware
 
 from opentelemetry import trace
-from opentelemetry.exporter.jaeger.thrift import JaegerExporter
+from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import OTLPSpanExporter
```

Comment:
Most recent [lint job](https://github.com/open-telemetry/opentelemetry-python/actions/runs/9811057144/job/27092540473) is asking to change this import:

```suggestion
from opentelemetry.exporter.otlp.proto.grpc.trace_exporter import (
    OTLPSpanExporter,
)
```